packages/block-library/src/index.js

```diff
@@ -242,35 +242,35 @@ export const registerCoreBlocks = (
  * __experimentalRegisterExperimentalCoreBlocks( settings );
  * ```
  */
-export const __experimentalRegisterExperimentalCoreBlocks =
-	process.env.GUTENBERG_PHASE === 2
-		? ( { enableFSEBlocks } = {} ) => {
-				[
-					// Experimental blocks.
-					homeLink,
+export const __experimentalRegisterExperimentalCoreBlocks = process.env
+	.IS_GUTENBERG_PLUGIN
```

Comment:
Seems like the linting rule doesn't like the way prettier breaks this onto two lines (I don't like much either!).

Probably because the rule doesn't account for line breaks and white space. I'll see if I can fix that.